### PR TITLE
Qiime 2 analysis help guide

### PR DESCRIPTION
**Pull request by adswafford**

### CODE REVIEW

**Comment by antgonza** [comment]
👍  once tests pass.

**Comment by antgonza** [comment]
Note that there are errors: https://travis-ci.org/biocore/qiita/jobs/256958027

**Comment by adswafford** [comment]
We created a new folder "analysisguide/" under "qiita_pet/support_files/doc/source/" with one file, "index.rst"; so I'm not sure what the error is unless there is an issue with adding a new folder?

File: qiita_pet/support_files/doc/source/index.rst [hunk 1]
```diff
@@ -3,34 +3,42 @@
...
+3) Easily interface with the EBI repository for automated deposition. Query and interact with Qiita data programmatically.
 
-The latests tutorials can be found in:
-`CMI Qiita/GNPS workshop pages <http://cmi-workshop.readthedocs.io/en/latest/>`__.
+Looking for information about our new analysis pipeline? Please see the document here:
+  
+.. toctree::
+   :maxdepth: 1  
+   
+   analysisguide/index.rst
```
Comment (antgonza) [inline qiita_pet/support_files/doc/source/index.rst]: the error is here: analysisguide/index as it should be: analysisguides/index.rst

**Comment by adswafford** [comment]
Oh weird, didn't even see that file in this pull request. Still getting
used to GitHub, so pardon my ineptitude. All of my software dev before here
has been solo :-/

I went to try to fix it in my patch, and it looks like you already have,
but I can't approve it. Any action needed on my end?

A

--
Austin D. Swafford, PhD
Director of Research, Center for Microbiome Innovation
Jacobs School of Engineering *| *UC San Diego

**Comment by antgonza** [comment]
I haven't done anything, I think @tkosciol will be able to help you faster.

**Comment by tkosciol** [comment]
Hi @adswafford !
I think what you need to do if you want to edit this online, is to go to `Files changed` section of this PR and edit the file that causes this error. This should update this specific PR.
Afterwards we should see it passing all checks and Antonio can review if he likes what we did.

**Comment by adswafford** [comment]
Okay, so what's confusing is that the file I see shows it as

"analysisguides/index.rst" not "analysisguides/index", so I don't
understand what to fix... You can call me after Code Review if you need to.

--
Austin D. Swafford, PhD
Director of Research, Center for Microbiome Innovation
Jacobs School of Engineering *| *UC San Diego

**Comment by antgonza** [comment]
I just fixed it.

**Comment by adswafford** [comment]
Cool. Thanks!

--
Austin D. Swafford, PhD
Director of Research, Center for Microbiome Innovation
Jacobs School of Engineering *| *UC San Diego

On Mon, Jul 24, 2017 at 11:12 AM, Antonio Gonzalez <notifications@github.com

**Comment by antgonza** [comment]
the failure is stochastic so merging.

**Pull request merged**